<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>415</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <widget class="QCheckBox" name="tabCheckBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>251</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Start new tab for search</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="ignoreCheckBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>90</y>
         <width>141</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Ignore file open errors</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cStyleCheckBox">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>120</y>
         <width>231</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Only search C-style function names</string>
       </property>
      </widget>
      <widget class="QDialogButtonBox" name="generalButtonBox">
       <property name="geometry">
        <rect>
         <x>330</x>
         <y>330</y>
         <width>156</width>
         <height>23</height>
        </rect>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Presets</string>
      </attribute>
      <widget class="QComboBox" name="targetComboBox">
       <property name="geometry">
        <rect>
         <x>64</x>
         <y>13</y>
         <width>421</width>
         <height>20</height>
        </rect>
       </property>
       <property name="editable">
        <bool>false</bool>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QLabel" name="targetLabel">
       <property name="geometry">
        <rect>
         <x>23</x>
         <y>13</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="maximumSize">
        <size>
         <width>232</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Target</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>40</y>
         <width>421</width>
         <height>20</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </widget>
      <widget class="QRadioButton" name="skipRadioButton">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>40</y>
         <width>136</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Skip</string>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="onlyRadioButton">
       <property name="geometry">
        <rect>
         <x>209</x>
         <y>40</y>
         <width>135</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Only</string>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QRadioButton" name="alsoRadioButton">
       <property name="geometry">
        <rect>
         <x>67</x>
         <y>40</y>
         <width>136</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Also</string>
       </property>
       <attribute name="buttonGroup">
        <string>buttonGroup</string>
       </attribute>
      </widget>
      <widget class="QLabel" name="moreEgLabel">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>80</y>
         <width>191</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>eg. -also type1 -also type2 -skip type3</string>
       </property>
      </widget>
      <widget class="QLabel" name="moreLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>100</y>
         <width>46</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>More</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="moreLineEdit">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>100</y>
         <width>421</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>-3</x>
         <y>150</y>
         <width>521</width>
         <height>20</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="typenameLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>180</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Typename</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="typenameLineEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>180</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="extensionsLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Extensions</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="extensionsLineEdit">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>240</y>
         <width>401</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="extensionsEgLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>220</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>eg. bat com exe</string>
       </property>
      </widget>
      <widget class="QPushButton" name="setTypePushButton">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>290</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Set type</string>
       </property>
      </widget>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="geometry">
        <rect>
         <x>265</x>
         <y>340</y>
         <width>241</width>
         <height>25</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Directories</string>
      </attribute>
      <widget class="QListWidget" name="includeListWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>161</width>
         <height>301</height>
        </rect>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="includeLabel">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>10</y>
         <width>46</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Include</string>
       </property>
      </widget>
      <widget class="QLabel" name="excludeLabel">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>10</y>
         <width>46</width>
         <height>14</height>
        </rect>
       </property>
       <property name="text">
        <string>Exclude</string>
       </property>
      </widget>
      <widget class="QListWidget" name="excludeListWidget">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>30</y>
         <width>161</width>
         <height>301</height>
        </rect>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="searchAllDirsCheckBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>340</y>
         <width>121</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Search all directories</string>
       </property>
      </widget>
      <widget class="QPushButton" name="defaultPushButton">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>240</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Default</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rightPushButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>130</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>-&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="leftAllPushButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>300</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rightAllPushButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>270</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="leftPushButton">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>31</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;-</string>
       </property>
      </widget>
      <widget class="QDialogButtonBox" name="directoriesButtonBox">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>340</y>
         <width>191</width>
         <height>25</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
      <widget class="QPushButton" name="deletePushButton">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>60</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
      <widget class="QPushButton" name="addPushButton">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>30</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Settings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Settings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
